### DISCUSSION TITLE
Autodig feature crashing up on joining servers

### DISCUSSION THREAD

**Discussion by Zippedy** [ORelio/Minecraft-Console-Client#2810] [Q&A]
Hi there, I have a problem with the most recent MCC. It lets me join servers, but when I try to join with the terrain and movements (I think) for the auto dig feature, the client crashes. I have a debug log of it, and I can't seem to find anything that would cause the crash. Any one have ideas as to what's wrong?

**Comment by Zippedy** [comment]
It lets me join the server with terrain and movement. But when I do the command /server skyblock it crashes. The server I want to use this feature on is a skyblock server that has a custom resource pack. Not sure if any of this helps solve the problem.

**Reply by milutinke** [reply]
What is the version of the client, what is the version of the server?
Attach the error message you get on the crash.
